src/reducers/MapReducer.js

```diff
@@ -36,6 +36,9 @@ export default (state = INITIAL_STATE, action) => {
         case MAP_REGION_CHANGE:
             return {...state, region: payload};
 
+        case FACEBOOK_LOGOUT_COMPLETE:
+            return INITIAL_STATE;
```

Comment:
This is a great example of the power of `redux`. When we logged out of facebook in `AuthActions`, we fired off the 
 `FACEBOOK_LOGOUT_COMPLETE` type. We were already handling in `AuthReducer` the removal of the token, but now by just adding a case-statement in `MapReducer` as well, we can also clear out any local map state. @amgoodman5 keep this pattern in mind... as we build out the app and other reducers, we'll need to have a `FACEBOOK_LOGOUT_COMPLETE` case in each of them to reset their portion of state as well to the initial defaults. For offline persisted data in `AsyncStorage`, `redux-persist` will handle it. But for anything that's still in memory (if a user logs out but doesn't force-quit or kill the app), we'll need to also manually clear the state for every reducer.

Reply:
nice, this is incredible!